### ISSUE TITLE
The app crashes when pressing share on the artist's page.

### ISSUE THREAD

**Issue by mostafaalagamy** [Malopieds/InnerTune#192]
### Checklist

- [X] I am able to reproduce the bug with the [latest debug version](https://github.com/z-huang/InnerTune/actions).
- [X] I've checked that there is no open or closed issue about this bug.
- [X] This issue contains only one bug.
- [X] The title of this issue accurately describes the bug.

### Steps to reproduce the bug

__

### Expected behavior

__

### Actual behavior

__

### Screenshots/Screen recordings

https://github.com/Malopieds/InnerTune/assets/80542861/5e1fdf11-84a0-400d-b1a4-ad4aba537dbe

### Logs

__

### InnerTune version

debug version

### Android version

13

**Comment by Malopieds** [comment]
Works fine for me. Does it works 6aca5a9ced062cef3dc465aea2df92b7324edf3f ?

Edit: **before** this commit

**Comment by mostafaalagamy** [comment]
> Works fine for me. Does it works 6aca5a9ced062cef3dc465aea2df92b7324edf3f ?
> 
> Edit: **before** this commit

I know, but the problem is on some devices, I'm testing all versions

**Comment by Malopieds** [comment]
Can you still reproduce this ?

**Comment by mostafaalagamy** [comment]
> Can you still reproduce this ?

Yep , Here and in the original version as well, it always happens when sharing an artist, and sometimes when sharing a song.
